Answers.md

```diff
@@ -0,0 +1,50 @@
+1. What is the differnce between Responsive & Adaptive website?
+
+	- I would say the primary difference is that an adaptive site
+	will change a few key features, such as a navigation bar or
+	changing text from a row to a column.
+
+	- A fully responsive site completely adjusts to fit as many
+	screens as it possibly can while still looking neat. It's
+	a lot more work but it's very worth it. As how we have been
+	setting up every page makes them very easy to modify
+
+
+2. Mobile first VS Desktop first
+
+	- Building something 'X'-first means that your original page
+	or file you start with. Is being made for that platform.
+	So building something desktop first is starting with the page
+	being made for a desktop user. vis-a-vers-a
+
+
+3. What does 'font-size: 62.5%' in the 'html' tag do?
+
+	- Something awesome. With using REM units, it makes the text 
+	displaying on said page look extrememly similar cross platform.
+
+
+4. How would I describe preprocessing w/o terminology?
+
+	- "Less is used to make writing CSS easier. CSS just by itself can
+	get very overwhelming and extremely hard to organize when the project
+	starts getting really big. So Less is used to write cleaner looking
+	and reading code that is just easier to read, then it reformats the
+	Less code into CSS code. I like it a lot."
+
+5. What is my favorite concept in Preprocessing? What gives me trouble?
+
+	- Preprocessing itself. I absolutely am enthralled to be using LESS.
+	After we went over this I feel very confident about building my own
+	web page for myself. The flow is really funky to think about at first.
+	You have a main.css file that is being live compiled from a index.less
+	file. Which in it's self has no styling, but imports from other 
+	stuff.less files. This setup can look daunting at first. But once
+	I looked over everything and realized how well everything is organized
+	using these files was very easy.
+
+	- The trouble I've had has come from me learning how to program and
+	organize my own files better. Before school I didn't realise how
+	"messy" my file system was. So I've been completely re-learning how to
+	name things and place files. But I'm happy to be figuring out much 
+	better ways of doing all this than before.
```

Comment:
Yea I agree. I had trouble with it as well. Especially because theres not everyone does it the same. But at the very least if you have things named well and everything in the same place it should work out well